libcxx/test/libcxx-03/numerics/rand/rand.device/has-no-random-device.verify.cpp

```diff
@@ -0,0 +1,18 @@
+//===----------------------------------------------------------------------===//
```

Comment:
If we don't have any corresponding test for other carve outs, I'd be OK with removing this test entirely.